### ISSUE TITLE
Hide controller rays when in orbit mode

### ISSUE THREAD

**Issue by chrisplatorres** [exokitxr/studio#5]
It makes sense to show the controller rays while in fake vr/VR mode, but hide them while in orbit controls/transform controls mode.

![raysorbitCapture](https://user-images.githubusercontent.com/29695350/59036376-c4074b00-8834-11e9-92b8-3b6cc913fe92.PNG)

**Comment by modulesio** [comment]
We should probably put in the actual controller models for the VR modes as well (A-Frame components have this detection feature).

**Comment by chrisplatorres** [comment]
Relevant to that I remember a recent WebXR talk mentioned having a CDN for that as well, couldn't find anything on it though except this part in the explainer:

```
The WebXR Device API currently does not offer any way to retrieve renderable resources that represent the input devices from the API itself, and as such the Gamepad's id must be used as a key to load an appropriate resources from the application's server or a CDN. The example below presumes that the getInputSourceRenderableModel call would do the required lookup and caching.
```
https://github.com/immersive-web/webxr/blob/393984aa56c7c419a0b3c68ffc6f89312aa1ae84/input-explainer.md#choosing-renderable-models

**Comment by modulesio** [comment]
☝️ Yes, that is how everyone currently does it.

**Comment by chrisplatorres** [comment]
fixed by #19
